source/appModules/vipmud.py

```diff
@@ -14,6 +14,8 @@
 This module makes NVDA read incoming text, as well as allowing the user to review the last nine messages with control 1 through 9.
 """
 
+category_title = "Vipmud"

```

Comment:
Idem for constant naming and translability.

Also regarding the application's name, I do not know this application and have not it on my side. But googling it, it seems that at least the "M" is upper case. Maybe MUD is all uppercase and maybe there is a space before "M"; to be checked.